File: KuantumKaos/python/python_kaos/anti_madde.py
from kuantum_nesnesi import KuantumNesnesi
from kritik import Kritik
from kuantum_cokusu_exception import KuantumCokusuException

class AntiMadde(KuantumNesnesi, Kritik):
    def analiz_et(self):
        self.stabilite -= 35
        print("Anti-madde tepkimeye giriyor!")

        if self.stabilite <= 0:
            raise KuantumCokusuException(self.id)

    def acil_durum_sogutmasi(self):
        self.stabilite += 50
        print("Anti-madde kararlılığı artırıldı.")

File: KuantumKaos/csharp/kuantumKaos/AntiMadde.cs
using System;

namespace kuantumKaos
{
    public class AntiMadde : KuantumNesnesi, IKritik
    {
        public override void AnalizEt()
        {
            Console.WriteLine("Evrenin dokusu titriyor...");

            Stabilite -= 25;

            if (Stabilite <= 0)
            {
                throw new KuantumCokusuException(ID);
            }
        }

        public void AcilDurumSogutmasi()
        {
            Stabilite += 50;

            if (Stabilite > 100)
                Stabilite = 100;

            Console.WriteLine("Anti madde acil soğutma uygulandı!");
        }
    }
}